dss/events/handlers/index.py

```diff
@@ -59,6 +59,44 @@ def process_new_indexable_object(bucket_name: str, key: str, replica: str, logge
         logger.debug(f"Not indexing {replica} creation event for key: {key}")
 
 
+def process_s3_removed_object(event, logger) -> None:
+    try:
+        # This function is only called for S3 removal events
+        key = unquote(event['Records'][0]["s3"]["object"]["key"])
+        process_removed_object(key, "aws", logger)
+    except Exception as ex:
+        logger.error("Exception occurred while processing S3 event: %s Event: %s", ex, json.dumps(event, indent=4))
+        raise
+
+
+def process_removed_object(key: str, replica: str, logger) -> None:
+    index_name = Config.get_es_index_name(ESIndexType.docs, Replica[replica])
+    bundle_id = get_bundle_id_from_key(key)
+    if bundle_is_indexed(index_name, bundle_id, logger):
+        logger.info(f"Received {replica} removal event for bundle which will be de-indexed: {key}")
+        ElasticsearchClient.get(logger).delete(index_name,
```

Comment:
nit. The Elasticsearch-py documentation [here](https://elasticsearch-py.readthedocs.io/en/master/api.html) recommends using keyword arguments for all calls for consistency and safety, and that that is (generally) what we have been doing.